a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -219,122 +219,187 @@ <h3>Check for Node</h3>
-					</dd>
-					<dt><var>non_textual_content_images</var></dt>
-					<dd>
-						<p>If true it indicates that at least one of the following is present in the package document:</p>
-						<ul>
-							<li><i>accessMode="chartOnVisual"</i> (Charts and Graphs);</li>
-							<li><i>accessMode="chemOnVisual"</i> (Chemistry);</li>
-							<li><i>accessMode="diagramOnVisual"</i> (Diagrams);</li>
-							<li><i>accessMode="mathOnVisual"</i> (Math);</li>
-							<li><i>accessMode="musicOnVisual"</i> (Music);</li>
-							<li><i>accessMode="textOnVisual"</i> (Images of text);</li>
-							<li>otherwise if false it means that this metadata is not present.</li>
-						</ul>
-						<p>This means that the content contains images of any type.</p>
-					</dd>
-					<dt><var>textual_alternative_images</var></dt>
-					<dd>
-						<p>If true it indicates that at least one of the following is present in the package document:</p>
-						<ul>
-							<li><i>accessibilityFeature="alternativeText"</i> (Short alternative textual descriptions);</li>
-							<li><i>accessibilityFeature="longDescription"</i> (Full alternative textual descriptions);</li>
-							<li><i>accessibilityFeature="describedMath"</i> (Visual of math fully described));</li>
-							<li>otherwise if false it means that this metadata is not present.</li>ff
-						</ul>
-						<p>This means that there are textual alternatives for images.</p>
-					</dd>
-				</dl>
-				<h4>Variables setup</h4>
-				<ol class="condition">
-					<li><b>LET</b> <var>package_document</var> be the result of calling <a href="#pre-processing">preprocessing</a> given <var>package_document_as_text</var>.</li>
+                    </ol>
+                    <h5>Instructions</h5>
+                    <ol class="condition">
+                        <li>
+                            <span><b>IF</b> <var>all_textual_content_can_be_modified</var>:</span>
+                            <span><b>THEN</b> display <code id="ways-of-reading-visual-adjustments-modifiable">"Appearance can be modified"</code>.</span>
+                        </li>
+                        <li>
+                            <span><b>ELSE IF</b> <var>is_fixed_layout</var>:</span>
+                            <span><b>THEN</b> display <code id="ways-of-reading-visual-adjustments-unmodifiable">"Appearance cannot be modified"</code>.</span>
+                        </li>
+                        <li><b>ELSE</b> display <code id="ways-of-reading-visual-adjustments-unknown">"No information is available"</code>.</li>
```

Comment:
These strings need to be made more specific. In this case, I think we were discussing using something like "No information about appearance modifiability is available."